client/src/components/History/CurrentHistory/HistoryFilters/HistoryFilters.test.js

```diff
@@ -1,7 +1,7 @@
 import { mount } from "@vue/test-utils";
-import { getLocalVue } from "tests/jest/helpers";
 import HistoryFilters from "./HistoryFilters";
-import { getQueryDict } from "utils/filterConversion";
+import { getLocalVue } from "tests/jest/helpers";
+import { HistoryFilters as HistoryFiltersUtil } from "components/History/HistoryFilters";
```

Comment:
Maybe here we can just avoid this naming conflict by calling 1) HistoryFilters(.vue) and the other 2) HistoryFiltering(.js) particularly since it is generated using `new Filtering()`. It still is a little confusing but less so I think.